Firestore/Example/Podfile

```diff
@@ -56,7 +56,8 @@ end
 # required.
 def configure_local_pods()
   # Firestore is always local; that's what's under development here.
-  pod 'FirebaseFirestore', :path => '../..'
+  pod 'FirebaseFirestoreInternal', :path => '../..'
```

Comment:
Why is this needed? Shouldn't the example work without explicitly importing `FirebaseFirestoreInternal`?

Reply:
So if you look at lines 66 – 69, you'll see that other Firestore dependencies are locally sourced. So this is just another dependency to grab locally rather than from a specs repo.

Reply:
I moved it down to be part of the 66 – 69 grouping.